### DISCUSSION TITLE
Wireguard UI

### DISCUSSION THREAD

**Discussion by Koliera** [tteck/Proxmox#1379] [General]
Hi, question - I remember some time ago the Wireguard LCX did have a nice web UI - why was it removed?
While the basal functions of the current LCX are great, it kind of feels like it might be good to have that other option that might give more graphical control. 
This seems like an interesting project with interesting functions - https://github.com/notthebee/ansible-easy-vpn
Would it be possible to mirror some of the functions?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/914
